### PR TITLE
DOC-176 Updated APIM and AM Helm charts installation readme's to advi…

### PR DESCRIPTION
**Pull request by ivan-gravitee**
DOC-176 Updated APIM and AM Helm charts installation readme's to advise on creating a specific namespace for each installation

**Jira ticket**

https://graviteedevops.atlassian.net/browse/DOC-176

**Description**

DOC-176 Updated APIM and AM Helm charts installation readme's to advise on creating a specific namespace for each installation

### CODE REVIEW

File: am/README.adoc [hunk 1]
```diff
@@ -12,38 +12,36 @@ This chart will deploy the following:
...
+Now, install the chart from the Helm repo with the release name
 
-....
-$ helm package .
-....
+`+graviteeio-am+`.
+
+To prevent potential issues in the future, it is best practice to create a separate namespace for your installation in order to prevent the use of the default Kubernetes namespace. The installation command provided immediately below assumes that such best practice is followed, however this is not a mandatory requirement.
+
+To install the Helm Chart using a dedicated namespace (we `+gravitee-am+` as an example), run the following command:
```
Comment (gaetanmaisse) [inline am/README.adoc:33]:
```suggestion
To install the Helm Chart using a dedicated namespace (we use `+gravitee-am+` as an example), run the following command:
```

**Review by gaetanmaisse — APPROVED** [review]
LGTM, just a minor typo feedback

**Comment by ivan-gravitee** [comment]
> LGTM, just a minor typo feedback

Good spot - thanks @gaetanmaisse! Pushed the change.

**Comment by ivan-gravitee** [comment]
Hey @Johngren - are you happy with this change?

**Review by brasseld — APPROVED** [review]

**Pull request merged**